toga/interface/dialogs.py

```diff
@@ -0,0 +1,22 @@
+def info_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def question_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def confirm_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def error_dialog(window, title, message):
+    raise NotImplementedError()
+
+
+def stack_trace_dialog(window, title, message, content, retry=False):
+    raise NotImplementedError()
+
+
+def save_file_dialog(window, title, suggested_filename, file_types):
```

Comment:
Is there scope to set a default suggested filename of "no suggestion", and file_types of "*"?

A default title might also be appropriate - although the default title may be different on each platform.